src/ParametricUtils.jl

```diff
@@ -138,20 +138,20 @@ function _totalCost(cfdict::Dict{Symbol, CalcFactorMahalanobis},
 end
 
 
-# build the cost function
+# DEPRECATED slower version
```

Comment:
Could we move this function to the Deprecated.jl file perhaps?

Reply:
I didn't because it's still in use and I'm not 100% sure in which direction to go. The overhead bothers me a bit and I think it could be better to add some parametric stuff to something like CCW.